### PR TITLE
Cert-csi documentation

### PR DESCRIPTION
**Pull request by spriya-m**
# Description
Documentation related to cert-csi tool.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#628 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Did you add the examples wherever applicable?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No edits are required.

**Review by satyakonduri — DISMISSED** [review]

**Review by tssushma — DISMISSED** [review]

**Review by rajkumar-palani — COMMENTED** [review]
1. Rename the images based on the functionalities
2. Follow Camelcase for all the images
3. Extensions of the images (png) should be either lowercase or uppercase and not both.
4. Do Local Preview to ensure document is properly loaded.

File: content/docs/csidriver/installation/test/certcsi.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+make install-nix
+```
+> Alternatively you can use instruction from https://github.com/kubernetes-incubator/metrics-server
+
+## How to Run
+
+To get information on how to use program you can use built-in help. 
+
+Or if you're using UNIX-like system and enabled _auto-completion feature_ while installing the tool, then you can use shell's
+built-in auto-completion to navigate through program's subcommands and flags interactively by just pressing TAB.
```
[the remark this answers is not in the document]
Comment (spriya-m) [reply content/docs/csidriver/installation/test/certcsi.md]: Done, have addressed all 5 comments. 

>> 4. Do Local Preview to ensure document is properly loaded
I have deployed and verified the document locally.

**Comment by Deepak-Ghivari** [comment]
Is it necessary to share the console output in the snippets/images? Console output can change anytime in future and we will have to keep on updating the images each time.

**Comment by Deepak-Ghivari** [comment]
listruns.png has FAILURE messages in the console logs? Is it expected?

**Comment by Deepak-Ghivari** [comment]
In few of the snippets/images the host details are masked. Consider masking it completely including "https://". Not a mandatory one but good and safe to have.

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No grammatical edits are required.

**Comment by spriya-m** [comment]
> 

IMO, it is fine to have failures. The image is just a representation of list of test runs, including failed runs.

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by shanmydell — APPROVED** [review]

**Pull request merged**